### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by SaadMahi**
Restaurant Web-Site using html, css and little Java.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+        <form action="backend.php">
+          <div class="name area">
+            Name<input type="text" name="" id="" placeholder="Enter your name" />
+          </div>
+          <div class="email area">
+            Email<input type="email" name="" id="" placeholder="Enter your email address" />
+          </div>
+          <div class="form area">
+            <p>Tell us your concern</p>
+            <textarea name="" id="" cols="30" rows="10" placeholder="Type your message here"></textarea>
```
Comment (OSAMA478) [inline index.html]: remove these empty unnecessary attributes (name and id)...

**Review by OSAMA478 — APPROVED** [review]
very nice...

File: index.html [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+      referrerpolicy="no-referrer"
+    />
+    <!--Style Sheet-->
+    <link rel="stylesheet" href="style.css" />
+    <link rel="icon" href="./images/logo.png">
+    <title>Zaatar-Restaurant</title>
+  </head>
+  <body>
+    <!--Step 1 Create a Nav Bar-->
+    <nav class="navbar">
```
Comment (murtazshaikh) [inline index.html]: generally put nav tag  inside header tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+        <ul class="menu-items">
+          <li><a href="#showcase">Home</a></li>
+          <li><a href="#about-us">About</a></li>
+          <li><a href="#food">Category</a></li>
+          <li><a href="#food-menu">Menu</a></li>
+          <li><a href="#customer">Testimonial</a></li>
+          <li><a href="#contact-us">Contact</a></li>
+        </ul>
+        <h1 class="logo"><img src="./images/logo.png" alt="Logo"></h1>
+      </nav>
```
Comment (murtazshaikh) [inline index.html]: don't use nav tag inside nav tag use it once or adjacent

File: index.html [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+            regional cuisines spanning the Arab world, from the Maghreb to the
+            Fertile Crescent and the Arabian Peninsula. These cuisines are
+            centuries old and reflect the culture of trading in baharat, herbs,
+            and foods.
+          </p>
+        </div>
+        <div class="about-img">
+          <img
+            src="./images/camel-meat.jpg"
+            alt=""
```
Comment (murtazshaikh) [inline index.html]: alt?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+        </div>
+      </div>
+    </section>
+
+    <section id="food">
+      <h1>Types of Food</h1>
+
+      <div class="food-container container">
+        <div class="food 1">
+          <img src="./images/zaatar.jpg" alt="" />
```
Comment (murtazshaikh) [inline index.html]: alt?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+
+    <section id="food-menu">
+      <h1>FOOD MENU</h1>
+
+      <div class="menu-wrapper container">
+        <div class="food-menu m1">
+          <div class="food-img">
+            <img
+              src="./images/Kebab-Halabi-675x450.jpg"
+              alt="Kebab Hallabi"
```
Comment (murtazshaikh) [inline index.html]: good job added alt here

**Review by murtazshaikh — APPROVED** [review]
Take care of mentioned things

**Comment by murtazshaikh** [comment]
Very nicely done

File: index.html (hunk 4 above)
Comment (SaadMahi) [reply index.html]: alt added

File: index.html (hunk 6 above)
Comment (SaadMahi) [reply index.html]: all empty alt filled

File: index.html (hunk 5 above)
Comment (SaadMahi) [reply index.html]: mentioned

File: index.html (hunk 2 above)
Comment (SaadMahi) [reply index.html]: you mean to say i should use header tag then within header tag i should make a container of nav tag and do my work? please tell me more.

File: index.html (hunk 3 above)
Comment (SaadMahi) [reply index.html]: you means to say i should use semantic tags only once, whether it's nav, main or footer?

File: index.html (hunk 1 above)
Comment (SaadMahi) [reply index.html]: removed empty attributes

**Pull request merged**